Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Vector Creations Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM phusion/baseimage:0.9.22
```

Comment:
May I ask why this is used as a base image? As there is only one process running in this image (Synapse), afaik there is no advantage in using the extra supervisor employed by phusion-baseimage. The official `ubuntu` ( or `debian`) base image will probably be enough.

Reply:
+1.  The ubuntu base image is better maintained, smaller in size, and in 10X broader usage.   It seems most of the things phusion is solving for are no longer issues.

Reply:
If an additonal opinion would be asked here, we're also running from `debian:jessie` in https://github.com/allmende/docker-synapse/blob/master/Dockerfile#L1 without issues.

Reply:
alpine can also be used as the base, see https://github.com/ptman/synapse-docker